modules/javafx.controls/src/test/java/test/javafx/scene/control/ContextMenuTest.java

```diff
@@ -636,4 +640,106 @@ private ContextMenu createContextMenuAndShowSubMenu() {
         assertEquals("Expected " + item1.getText() + ", found " + focusedItem.getText(),
                 item1, focusedItem);
     }
+
+    @Test public void test_position_showOnScreen() {
+        ContextMenu cm = createContextMenu(false);
+        cm.show(anchorBtn,100, 100);
+
+        assertEquals(cm.getAnchorX(), 100, 0.0);
+        assertEquals(cm.getAnchorY(), 100, 0.0);
```

Comment:
The expected and actual values are backwards (expected should be first).

Reply:
Swapped positions.